code/modules/antagonists/bloodsuckers/powers/_powers.dm

```diff
@@ -40,6 +40,8 @@
 	var/constant_bloodcost = 0
 	///If the Power has any additional descriptions coming from either 3rd partys or the power itself
 	var/additional_text = ""
+	///Path to the ascended version of the power. For Lasombra bloodsucker powers
+	var/ascended_power = null
```

Comment:
This should work like Tremere does and be its own subtype I think.
That was one of the main reasons for the datumization of Bloodsucker Clans.

Reply:
if i don't do this soonish i'll do it in a followup pr